### PR TITLE
Watcher configs

### PR DESCRIPTION
**Pull request by dmitrydao**
## Overview

Whitespaces in `docker-compose-watcher.yml` file throw validation errors during `docker-compose -f docker-compose-watcher.yml` up function.

```
ERROR: yaml.scanner.ScannerError: while scanning for the next token
found character '\t' that cannot start any token
  in "./docker-compose-watcher.yml", line 18, column 11
```

## Changes

Removed unnecessary whitespaces in `docker-compose-watcher.yml` file.

P.S. I previously opened the same PR but accidentally removed the branch on my laptop, so I couldn't check why one of the tests wasn't passing.

### CODE REVIEW

File: docker-compose-watcher.yml [hunk 1]
```diff
@@ -15,39 +15,39 @@ services:
...
-      - CONTRACT_ADDRESS_PAYMENT_EXIT_GAME=0x960ca6b9faa85118ba6badbe0097b1afd8827fac	
-      - DATABASE_URL=postgres://omisego_dev:omisego_dev@postgres:5432/omisego_dev	
-      - PORT=7434	
-      - DD_DISABLED=true	
-      - DB_PATH=/app/.omg/data	
-      - ETHEREUM_EVENTS_CHECK_INTERVAL_MS=8000	
-      - ETHEREUM_STALLED_SYNC_THRESHOLD_MS=300000	
-      - ETHEREUM_BLOCK_TIME_SECONDS=15	
-      - EXIT_PROCESSOR_SLA_MARGIN=5520	
+    image: omisego/watcher:4445aee
```
Comment (unnawut) [inline docker-compose-watcher.yml:20]: Not to do with your PR, but if I could sneak this latest version in (should already be compatible with `*.ropsten.v1.omg.network`

```suggestion
    image: omisego/watcher:v0.4.8
```

Also applies to watcher_info below

Comment (dmitrydao) [reply docker-compose-watcher.yml:20]: Hmmm. I'm doing this from master branch. Maybe it's time to update the version there?

**Comment by dmitrydao** [comment]
It doesn't pass again on `watcher_coveralls_and_integration_tests` check. Can somebody tell what is the issue? CircleCI page redirects me to an unknown page for some reason.

**Comment by boolafish** [comment]
tests are a bit flaky now 😢 help you clicked the re-run btn

**Pull request merged**